test/functional/app/api/test_kyc_application_api.py

```diff
@@ -1,6 +1,9 @@
 import json, pytest
+import logging
 
+logg = logging.getLogger(__name__)
 
+@pytest.mark.xfail(reason='test uses wrong parameter for account type against the api, when replaced with correct the test fails')
```

Comment:
Not sure if we should be just xfailing thing rather than working out what's gone wrong?
I'll have a quick squiz but @tristanhcole u might be better placed to diagnose the situation here?

Reply:
Hmmm yeah i don't know enough about how this api works, but seems to be something to do with `user_id` only being needed if the user is an admin adding another user...?

Reply:
yeah I think it's a bug... should add this code to the KYC API. It needs to be rebuilt as logic is messy

https://github.com/teamsempo/SempoBlockchain/blob/21f89aa4d0ff9e5f0f2a0e2a89b7a9bdf28060a2/app/server/api/kyc_application_api.py#L304-L306

Reply:
There's 3 flows currently for the api
1. Mobile User KYC
2. Web KYC flow for an organisation as a superadmin
3. Web KYC flow for a user as another admin user